app/views/layouts/application.html.erb

```diff
@@ -3,18 +3,32 @@
 <head>
   <title>LaundryTime</title>
   <%= csrf_meta_tags %>
+  <%= stylesheet_link_tag "application.scss" %>
+  <style>
+    body {
+      background-color: rgb(204, 255, 253);
+      margin-top: 50px;
+      color: #633472;
+    h1 {
+      text-align: center;
+    }
+
+    h2 {
+      text-align: left;
+    }
+
+  </style>
```

Comment:
Your styling should be added to the scss file. Not the template.

Reply:
ok I move it